### PR TITLE
Update Repo Readme

### LINKED ISSUE #3177: Make translation instructions easier to find
Currently, the only locations where a link to instructions about how to help translate are the "Help Translate" button in the Scratch Addons settings and within the docs themselves. This makes it relatively hard to find and even harder to stumble across.

A couple of potential places to put it:
- Under the language dropdown on scratchaddons.com
- In the description for "International" (under the "Features" header) on scratchaddons.com
- In the footer on scratchaddons.com
- In the "Contribute" section of the README in the ScratchAddons repo
- In the "Contributing guidelines" (`CONTRIBUTING.md`) in the ScratchAddons repo
- Some/all of the above

### LINKED ISSUE #5411: Remove mention of persistent scripts from README
I know that Manifest V3 is going to make persistent scripts harder to manage, and that you removed the docs page on creating them earlier this year, but they're still mentioned in this repository's `README.md`. Should we remove the mention of them there too?

### PR DESCRIPTION
**Pull request by Samq64**
Resolves #3177 
Resolves #5411 
Closes #5603 

### Changes

- All of the current changes from #5603 
- Added official (except Edge) extension store badges replacing the user count ones
- Added a demo screenshot
- Slightly shorted the what is an addon sections and pointed to the docs
- Removed mentions of the Devtools extension and persistent scripts
- Removed all installation methods except git and linked the installation docs
- Added more contributing options (I think it needs more work)
- Added a list of all the top-level folders and what they are used for.
- Linked Scratch Addons in the first paragraph to the org
- Mention VSCode as the recommended editor

### Reason for changes

The README needs an update. I was going to merge some of my changes into #5603 but I think there are too many now.

### Tests

https://github.com/Samq64/ScratchAddons/tree/update-readme#readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,32 @@
...
+
+[![](https://img.shields.io/github/license/ScratchAddons/ScratchAddons?style=flat-square)](https://github.com/ScratchAddons/ScratchAddons/blob/master/LICENSE)
+[![](https://img.shields.io/badge/chat-on_discord-7289da.svg?style=flat-square)](https://discord.gg/R5NBqwMjNc) [![](https://img.shields.io/badge/website-scratchaddons.com-ff7b26.svg?style=flat-square)](https://scratchaddons.com)
+
+<img src="images/demo.png" alt="Scratch Addons demo" height=500px></img>
 
 ## About
 
-Scratch Addons is a WebExtension (supports both Chrome and Firefox). Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable.
+[Scratch Addons](https://github.com/ScratchAddons) combines new and existing features and themes for the Scratch website and project editor into one easy-to-access and configurable browser extension. The mission is to provide a centralized, up-to-date platform for community development of new features and themes for Scratch.
```
Comment (Samq64) [inline README.md:17]: I know this isn't a very good place to link to the rest of the org. Does anyone have any other ideas.

File: README.md [hunk 2]
```diff
@@ -43,41 +38,70 @@ Scratch Addons by itself is just an addon loader. Its main tasks are:
...
+
+- `.github`: GitHub templates, workflows and contributing files.
+- `_locales`: Translations for the settings page, popup and everything else except addons.
+- `addon-api`: The code for the `addon-*` APIs.
+- `addons-l10n`: Translations for strings userscripts add to a page. Each addon has its own file.
+- `addons`: The addons themselves. Each addon has it own directory with an `addon.json` and an entry in `addons.json`.
+- `background`: Background scripts for handling things such as addon settings.
+- `content-scripts`: The main extension script and addon injector.
+- `images`: Logos, screenshots and icons. Images only used by addons go in that addon's directory instead.
+- `libraries`: Libraries for things such as handling the global theme. There are also some third-party libraries.
```
Comment (Samq64) [inline README.md:53]: I'm not that familiar with these parts of the extension. Could someone else come up with better explanations?

File: images/badges/edge-badge.svg
Comment (Samq64) [inline images/badges/edge-badge.svg:1]: Note that this Edge badge is unofficial. The only official one is the "Get it from Microsoft" one and it looks like it has a bunch of terms.

**Comment by Hans5958** [comment]
I'd rather have the "About" section to be moved to the docs, letting the install section to have more visibility, where the big badges can be put there instead (e.g. https://scratchaddons.com/docs/getting-started/installing/#from-extension-stores, but simpler since we have badges).

File: README.md [hunk 3]
```diff
@@ -1,37 +1,32 @@
...
-[![](https://img.shields.io/amo/users/scratch-messaging-extension?style=flat-square&logo=firefox-browser&logoColor=white&label=users&color=FF7139)](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/)
-[![](https://img.shields.io/github/v/release/ScratchAddons/ScratchAddons?style=flat-square&logo=github&logoColor=white&label=version&color=181717)](https://github.com/ScratchAddons/ScratchAddons/releases)
-[![](https://img.shields.io/github/downloads/ScratchAddons/ScratchAddons/total?style=flat-square&logo=github&logoColor=white&label=downloads&color=181717)](https://github.com/ScratchAddons/ScratchAddons/releases)
-[![](https://img.shields.io/badge/discuss-on_github-181717.svg?style=flat-square)](https://github.com/ScratchAddons/ScratchAddons/discussions)
-[![](https://img.shields.io/badge/chat-on_discord-7289da.svg?style=flat-square)](https://discord.gg/R5NBqwMjNc)
-[![](https://img.shields.io/badge/website-scratchaddons.com-ff7b26.svg?style=flat-square)](https://scratchaddons.com)
+# Scratch Addons browser extension
```
Comment (Hans5958) [inline README.md:2]: Also, this has already been mentioned before (see last issue for further details), but I prefer this.

```suggestion
# Scratch Addons
```

File: README.md [hunk 4]
```diff
@@ -1,37 +1,32 @@
...
+
+![](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=version&color=E23A2E) ![](https://img.shields.io/amo/v/scratch-messaging-extension?style=flat-square&logo=firefox-browser&logoColor=white&label=version&color=FF7139) ![](https://img.shields.io/badge/dynamic/json?style=flat-square&logo=microsoftedge&label=version&prefix=v&color=067FD8&query=%24.version&url=https%3A%2F%2Fmicrosoftedge.microsoft.com%2Faddons%2Fgetproductdetailsbycrxid%2Filiepgjnemckemgnledoipfiilhajdjj) [![](https://img.shields.io/github/v/release/ScratchAddons/ScratchAddons?style=flat-square&logo=github&logoColor=white&label=version&color=181717)](https://github.com/ScratchAddons/ScratchAddons/releases)
+
+[![](https://img.shields.io/github/license/ScratchAddons/ScratchAddons?style=flat-square)](https://github.com/ScratchAddons/ScratchAddons/blob/master/LICENSE)
+[![](https://img.shields.io/badge/chat-on_discord-7289da.svg?style=flat-square)](https://discord.gg/R5NBqwMjNc) [![](https://img.shields.io/badge/website-scratchaddons.com-ff7b26.svg?style=flat-square)](https://scratchaddons.com)
```
Comment (Hans5958) [inline README.md:11]: New lines are used as a substitute for space so it doesn't form a long line.

```suggestion
[![](https://img.shields.io/github/license/ScratchAddons/ScratchAddons?style=flat-square)](https://github.com/ScratchAddons/ScratchAddons/blob/master/LICENSE)
[![](https://img.shields.io/badge/chat-on_discord-7289da.svg?style=flat-square)](https://discord.gg/R5NBqwMjNc)
[![](https://img.shields.io/badge/website-scratchaddons.com-ff7b26.svg?style=flat-square)](https://scratchaddons.com)
```

File: README.md [hunk 5]
```diff
@@ -1,37 +1,32 @@
...
+# Scratch Addons browser extension
+
+[![Chrome Web Store](images/badges/cws-badge.png)](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco)
+[![Firefox Add-ons](images/badges/ff-addon-badge.png)](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco)
+[![Edge Add-ons](images/badges/edge-badge.svg)](https://microsoftedge.microsoft.com/addons/detail/iliepgjnemckemgnledoipfiilhajdjj)
+
+![](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=version&color=E23A2E) ![](https://img.shields.io/amo/v/scratch-messaging-extension?style=flat-square&logo=firefox-browser&logoColor=white&label=version&color=FF7139) ![](https://img.shields.io/badge/dynamic/json?style=flat-square&logo=microsoftedge&label=version&prefix=v&color=067FD8&query=%24.version&url=https%3A%2F%2Fmicrosoftedge.microsoft.com%2Faddons%2Fgetproductdetailsbycrxid%2Filiepgjnemckemgnledoipfiilhajdjj) [![](https://img.shields.io/github/v/release/ScratchAddons/ScratchAddons?style=flat-square&logo=github&logoColor=white&label=version&color=181717)](https://github.com/ScratchAddons/ScratchAddons/releases)
```
Comment (Hans5958) [inline README.md:8]: Along with this, I'd rather have the links kept. It won't have much of a difference.

```suggestion
![](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=version&color=E23A2E)
![](https://img.shields.io/amo/v/scratch-messaging-extension?style=flat-square&logo=firefox-browser&logoColor=white&label=version&color=FF7139)
![](https://img.shields.io/badge/dynamic/json?style=flat-square&logo=microsoftedge&label=version&prefix=v&color=067FD8&query=%24.version&url=https%3A%2F%2Fmicrosoftedge.microsoft.com%2Faddons%2Fgetproductdetailsbycrxid%2Filiepgjnemckemgnledoipfiilhajdjj) 
[![](https://img.shields.io/github/v/release/ScratchAddons/ScratchAddons?style=flat-square&logo=github&logoColor=white&label=version&color=181717)](https://github.com/ScratchAddons/ScratchAddons/releases)
```

File: README.md [hunk 6]
```diff
@@ -43,41 +42,70 @@ Scratch Addons by itself is just an addon loader. Its main tasks are:
...
+
+Before contributing code, please read our [contributing guidelines](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CONTRIBUTING.md).
+
+Follow the installation instructions above to setup the extension. Please only submit pull requests for open issues and test your changes.
+
+We recommend Visual Studio Code as the code editor.
+
+### Translations
+
+Translations are handled by Trasfix. If you are interested in translating the extension, read [Joining the Localization Team](https://scratchaddons.com/docs/localization/joining-the-localization-team/)
```
Comment (DNin01) [inline README.md:104]:
```suggestion
Translations are handled by Trasifex. If you are interested in translating the extension, read [Joining the Localization Team](https://scratchaddons.com/docs/localization/joining-the-localization-team/)
```

File: README.md [hunk 7]
```diff
@@ -43,41 +42,70 @@ Scratch Addons by itself is just an addon loader. Its main tasks are:
...
-### From source
+For other installation methods and browser support, check [the documentation](https//scratchaddons.com/docs/getting-started/installing).
 
-No build is required. You may download the source code by doing one of these steps.
+### Load the extension (Chrome)
 
-1. Cloning the repository by running `git clone https://github.com/ScratchAddons/ScratchAddons.git`.
-2. Download the zipball of the current state of the repository [here](https://github.com/ScratchAddons/ScratchAddons/archive/master.zip).
-3. Download one of the builds on [the releases page](https://github.com/ScratchAddons/ScratchAddons/releases). (.zip recommended)
+Use this method on Microsoft Edge, Opera, Brave, Vivaldi and other Chromium-based browsers.
```
Comment (DNin01) [inline README.md:74]:
```suggestion
Use this method on Google Chrome, Microsoft Edge, Opera, Brave, Vivaldi and other Chromium-based browsers.
```

**Review by DNin01 — CHANGES_REQUESTED** [review]
I like how these changes are going, but a few things...

File: README.md (hunk 1 above)
Comment (DNin01) [reply README.md:21]: I don't know if we even need to link to the org.

Comment (Samq64) [reply README.md:21]: WorldLanguages had a todo on the other PR to link to the org readme, but I don't think this counts.

**Review by DNin01 — APPROVED** [review]
I think this is good enough!

File: README.md (hunk 3 above)
Comment (WorldLanguages) [reply README.md:2]: In the other PR, 4 people have said they prefer to keep "browser extension": https://github.com/ScratchAddons/ScratchAddons/pull/5603#issuecomment-1383153748

**Comment by Samq64** [comment]
> I'd rather have the "About" section to be moved to the docs

Were you thinking of removing the entire about section or just a couple subsections? The only one I would remove is Addons outside Scratch Addons.

File: README.md (hunk 3 above)
Comment (Hans5958) [reply README.md:2]: By the way, it can be interpreted to either that or "it doesn't matter, anything works".

Comment (Samq64) [reply README.md:2]: I normally wouldn't mind this, but because of https://github.com/ScratchAddons/ScratchAddons/pull/5603#issuecomment-1382080902 I do.

**Comment by Hans5958** [comment]
> > I'd rather have the "About" section to be moved to the docs
> 
> Were you thinking of removing the entire about section or just a couple subsections? The only one I would remove is addons outside Scratch Addons and moving the screenshot down.

I'd rather have the whole section removed from the README and put it back in (a new page of) the docs.

**Comment by Samq64** [comment]
> I'd rather have the whole section removed from the README and put it back in (a new page of) the docs.

Even if that happens, I think the first paragraph should stay.

**Comment by Hans5958** [comment]
> > I'd rather have the whole section removed from the README and put it back in (a new page of) the docs.
> 
> Even if that happens, I think the first paragraph should stay.

Oh, right. The mission statement can stay (the heading can also be removed but anything goes)

**Comment by Samq64** [comment]
I removed a couple subcategories and moved the screenshot and big badges down. Is that what you meant or did you want the small badges in the installation section?

**Comment by Secret-chest** [comment]
> The only one I would remove is addons outside Scratch Addons and moving the screenshot down.
I believe it's still important that users know TurboWarp uses your addons.

**Review by Hans5958 — CHANGES_REQUESTED** [review]
Let's move the badge images inside the `.github` folder, since it is only being used on the README. This should exclude it from being exported ("Download ZIP" on GitHub), just like the README.

<details>
<summary>PS. @Secret-chest</summary>

PS. @Secret-chest, I have edited your comment regarding the usage of blockquotes. You have to add an additional newline after the blockquote, or your reply will be conjoined with the quote

## ✔️

```
> What is it?
It is a chest.
```

> What is it?
It is a chest

## :x:

```
> What is it?

It is a chest.
```

> What is it?

It is a chest
</details>

**Comment by WorldLanguages** [comment]
Reminder: there's a related PR which suggests changes to the README #6051

**Comment by Samq64** [comment]
Prettier is driving me crazy right now!

**Comment by Samq64** [comment]
Now it decides to cooperate? Well I've already made the other PR without all the merge commits.

**Comment by Samq64** [comment]
Now it decides to cooperate? Well I've already made the other PR without all the merge commits.

**Pull request closed without merging**